pennylane_lightning/core/src/simulators/lightning_qubit/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ add_library(lightning_qubit STATIC ${LQUBIT_FILES})
 option(ENABLE_BLAS "Enable BLAS" OFF)
 option(ENABLE_GATE_DISPATCHER "Enable gate kernel dispatching on AVX/AVX2/AVX512" ON)
 option(LQ_ENABLE_KERNEL_OMP "Enable OpenMP pragmas for gate kernels" OFF)
+option(LQ_ENABLE_KERNEL_AVX_STREAMING "Enable AVX2/512 streaming operations for gate kernels" OFF)
```

Comment:
Do we want to set that `OFF` and issue a warning, or throw if `LQ_ENABLE_KERNEL_AVX_STREAMING=ON` but `LQ_ENABLE_KERNEL_OMP=OFF`?

Reply:
Good call. Added a warning